clarity.d.ts

```diff
@@ -187,8 +187,9 @@ interface ICompressedBatchMessage extends IWorkerMessage {
 type NumberJson = Array<number | number[]>;
 
 interface IShadowDomNode extends HTMLDivElement {
-  node: Node; /* Reference to the node in the real DOM */
-  ignore: boolean;  /* Flag to avoid sending data for that node */
+  node: Node;
+  info: INodeInfo;
+  updateInfo: () => INodeInfo;
```

Comment:
>updateInfo: () => INodeInfo; [](start = 2, length = 28)

What's the purpose of this function?

Reply:
Gets an updated state of the node, including the new state

---
In reply to: [182825081](https://github.com/Microsoft/clarity-js/pull/80#discussion_r182825081) [](ancestors = 182825081)